contents/product-engineers/best-customer-retention-metrics.md

```diff
@@ -0,0 +1,246 @@
+
+### How is it useful?
+
+This is another really important metric for subscription-based businesses. That means SaaS and e-commerce, but also bricks and mortar enterprises like hair salons and gyms. A strong understanding of CLTV will help identify high-value customers and enable you to build the best possible products and services for them.
+
+A clear understanding of your CLTV also gives you a clear understanding of who your most valuable customers are and how they behave. You can then, for example, work to move more people into that cohort, or work out how to better serve your most loyal customers.
+
+### How to calculate your CLTV
+
+Take the average revenue per user (ARPU) within a specific timeframe, say monthly or annually, and multiply it by your average customer lifespan. 
+
+For example, if your ARPU is $50 per month and your average customer lifespan is 12 months, then your CLTV would be $50 x 12, or $600.
+
+### What’s a good CLTV?
+
+To work this out you need to know your CAC. That’s your customer acquisition cost, or, simply put, the average cost to your business of snagging a new paying customer. If your CAC is lower than your CLTV then you’ve got a healthy business that’s able to acquire new paying customers without losing money. If it’s not, you're in trouble.
+
+As a guide, at early-stage startups the CLTV might be lower as you acquire new customers and find product-market fit. According to data compiled by search optimization firm [First Page Sage](https://firstpagesage.com/seo-blog/the-ltv-to-cac-ratio-benchmark/), the average CLTC to CAC ratio at B2B SaaS companies is 4:1 At B2C SaaS companies it’s 2.5:1.
+
+And this kind of healthiness compounds: acquiring customers without spending a ton on marketing allows you to focus more on improving and expanding your existing product, delighting your most loyal customers, and improving overall customer satisfaction.
+
+## 3. Revenue churn rate
+
+### What is revenue churn rate?
+
+It’s not just customers who churn, revenue churns with them. This metric helps you understand how much money your business makes from each user over time. It can help you spot where you’re losing revenue due to downgrades, people failing to renew, or, worst of all, cancellations. 
+
+While your customer churn rate looks at the number of customers you lose in a given period, your revenue churn specifically focuses on the dollar value of recurring period lost, or gained, in a certain period.
+
+### How is it useful?
+
+Any subscription-based business should be measuring their revenue churn rate. These days, that’s pretty much all businesses from coffee subscription businesses to B2B SaaS companies and everything in between. Businesses that sell big, high-value items (think houses, industrial equipment) likely don’t need to focus on revenue churn.
```

Comment:
```suggestion
It’s not just customers who churn, revenue churns with them. 

While your customer churn rate looks at the *number* of customers you lose in a given period, revenue churn specifically focuses on the *dollar value* of recurring revenue lost, or gained, in a certain period.

### Why is it useful?

Because it helps you understand how much money your business makes from each user over time, and the impact of customer churn on revenue.

Not all revenue churn comes from customers failing to renew, or cancelling their contract. It can also come from customers *downgrading* their subscription, or using your product less.

This is really useful for understanding the exact nature of retention and churn in your product. A company, for example, could have a low customer churn rate, but high revenue churn. This would suggest either:

1. Only high-value customers are churning.
2. Customers are choosing to spend less / downgrade, rather than churn entirely.

Thus, analyzing customer retention and revenue churn together is essential to finding the truth about your business.
```